code/game/vehicles/spacepods/equipment.dm

```diff
@@ -186,3 +187,41 @@
 /obj/item/device/spacepod_equipment/cargo/ore/removed(var/mob/user)
 	. = ..()
 	unload()
+
+/obj/item/device/spacepod_equipment/lock
+	name = "pod lock"
+	desc = "You shouldn't be seeing this"
+	icon = 'icons/goonstation/pods/ship.dmi'
+	icon_state = "blank"
+	var/mode = 0
+	var/id = null
+
+// Key and Tumbler System
+/obj/item/device/spacepod_equipment/lock/keyed
+	name = "\improper spacepod tumbler lock"
+	desc = "A locking system to stop podjacking. This version uses a standalone key."
+	icon_state = "pod_locator"
+
+/obj/item/device/spacepod_equipment/lock/keyed/New()
+	..()
+	id = rand(1, 99999)
```

Comment:
Clever, but mechanically will never be used ever (outside of maybe an event, in which case the keys can still be vv'd)
(the "never used ever" is an argument for not making the fix too, as the error itself will be rare and the chance the keys will be used on the lock they erroneously match is whew, it's very slim but it's only like 20 seconds work to get unique keys so, /shrug)